### PR TITLE
Fix SRTCP replay protection parameter

### PR DESCRIPTION
**Pull request by at-wat**
Maximum index value was wrong.
Add test cases to check uppper boundary of SRTP sequence number and SRTCP index.

### CODE REVIEW

**Comment by jeremija** [comment]
LGTM as far as I can tell (don't have a lot of context :)

File: srtcp_test.go [hunk 1]
```diff
@@ -12,36 +12,76 @@ import (
...
+	},
+	{
+		ssrc:      0x11111111,
+		index:     0x7ffffffe, // Upper boundary of index
+		encrypted: []byte{0x80, 0xc8, 0x00, 0x06, 0x11, 0x11, 0x11, 0x11, 0x17, 0x8c, 0x15, 0xf1, 0x4b, 0x11, 0xda, 0xf5, 0x74, 0x53, 0x86, 0x2b, 0xc9, 0x07, 0x29, 0x40, 0xbf, 0x22, 0xf6, 0x46, 0x11, 0xa4, 0xc1, 0x3a, 0xff, 0x5a, 0xbd, 0xd0, 0xf8, 0x8b, 0x38, 0xe4, 0x95, 0x38, 0x5d, 0xcf, 0x1b, 0xf5, 0x27, 0x77, 0xfb, 0xdb, 0x3f, 0x10, 0x68, 0x99, 0xd8, 0xad, 0xff, 0xff, 0xff, 0xff, 0x5a, 0x99, 0xce, 0xed, 0x9f, 0x2e, 0x4d, 0x9d, 0xfa, 0x97},
+		decrypted: []byte{0x80, 0xc8, 0x0, 0x6, 0x11, 0x11, 0x11, 0x11, 0x4, 0x99, 0x47, 0x53, 0xc4, 0x1e, 0xb9, 0xde, 0x52, 0xa3, 0x1d, 0x77, 0x2f, 0xff, 0xcc, 0x75, 0xbb, 0x6a, 0x29, 0xb8, 0x1, 0xb7, 0x2e, 0x4b, 0x4e, 0xcb, 0xa4, 0x81, 0x2d, 0x46, 0x4, 0x5e, 0x86, 0x90, 0x17, 0x4f, 0x4d, 0x78, 0x2f, 0x58, 0xb8, 0x67, 0x91, 0x89, 0xe3, 0x61, 0x1, 0x7d},
+	},
+	{
+		ssrc:      0x11111111,
+		index:     0x7fffffff, // Will be wrapped to 0
```
Comment (jeremija) [inline srtcp_test.go:43]: I can't tell where can this be seen in the test? (I believe you, I'd just like to learn :)

Comment (at-wat) [reply srtcp_test.go:43]: It's in `len()-14` to `len()-10` bytes of 'encrypted' bytes `0x80, 0x00, 0x00, 0x00` (first bit is an encrypted flag.)

Comment (at-wat) [reply srtcp_test.go:43]: 10-bytes from the end are the auth tag. 4-bytes before this is encrypted flag (`0x80000000`) and SRTCP index (max `0x7FFFFFFF`).

Comment (jeremija) [reply srtcp_test.go:43]: Got it, thanks for explaining!
